blockchain/fullblocktests/generate.go

```diff
@@ -230,6 +230,19 @@ func payToScriptHashScript(redeemScript []byte) []byte {
 	return script
 }
 
+// payToScriptHashScript32 returns a standard pay-to-script-hash for the provided
+// redeem script.
+func payToScriptHashScript32(redeemScript []byte) []byte {
+	redeemScriptHash := bchutil.Hash256(redeemScript)
```

Comment:
I think this is another place that needs the updated `bchutil`. Leaving a note to come back and check when that is available.

Reply:
not available yet in the branch I have from opreturn's fork of bchutil.